presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraMetadata.java

```diff
@@ -311,4 +313,28 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
     {
         return Optional.empty();
     }
+
+    @Override
+    public ConnectorInsertTableHandle beginInsert(ConnectorSession session, ConnectorTableHandle tableHandle)
+    {
+        CassandraTableHandle table = (CassandraTableHandle) tableHandle;
+        String schemaName = cassandraSession.getCaseSensitiveSchemaName(table.getSchemaName());
```

Comment:
No need to do `getCaseSensitiveSchemaName`. It is case sensitive in `CassandraTableHandle` already